lib/pharos/addons/open_ebs.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Addons
+    class OpenEbs < Pharos::Addon
+      name 'openebs'
+      version '0.5.3'
+      license 'Apache License 2.0'
+
+      struct {
+        attribute :default_replicas, Pharos::Types::Int
+        attribute :default_capacity, Pharos::Types::String
+        attribute :default_class, Pharos::Types::Bool
+        attribute :default_storage_path, Pharos::Types::String
+      }
+
+      schema {
+        optional(:default_replicas).filled(:int?)
```

Comment:
I don't like repetition of `default_`. These are all `StorageClass` options?

Reply:
Yes, they're mostly for the "default" `StorageClass` that gets created during the add-on setup. `default_replicas`, but I _think_ it's a value that gets used if the storage class does not provide amount of replicas.

Would it make more sense to move these to something like:
```
openebs:
  enabled: true
  storage_class:
    size: 50G
    replicas: 3
    ...
```
??